test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -137,6 +144,27 @@ func (config *KubeletManagedHostConfig) getEtcHostsContent(podName, containerNam
 }
 
 func (config *KubeletManagedHostConfig) createPodSpec(podName string) *v1.Pod {
+
+	etcHostContents := `127.0.0.1	localhost
```

Comment:
seems like creation of this file should be done in setup/cleanup, not inside createPodSpec (which could be called multiple times, or in an unexpected order, in the future)

Reply:
done